content/docs/guides/cli.md

```diff
@@ -16,6 +16,75 @@ weight: 1
 Download platform specific compressed package from the [Releases page](https://github.com/openservicemesh/osm/releases).
 Unpack the `osm` binary and add it to `$PATH` to get started.
 
+#### Linux and macOS
+
+In a bash-based shell on Linux/macOS or [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/about), use `curl` to download the OSM release and then extract with `tar` as follows:
+
+```console
+# Specify the OSM version that will be leveraged throughout these instructions
+OSM_VERSION=v0.9.1
+
+# Linux curl command only
+curl -sL "https://github.com/openservicemesh/osm/releases/download/$OSM_VERSION/osm-$OSM_VERSION-linux-amd64.tar.gz" | tar -vxzf -
+
+# macOS curl command only
+curl -sL "https://github.com/openservicemesh/osm/releases/download/$OSM_VERSION/osm-$OSM_VERSION-darwin-amd64.tar.gz" | tar -vxzf -
+```
+
+The `osm` client binary runs on your client machine and allows you to manage OSM in your AKS cluster. Use the following commands to install the OSM `osm` client binary in a bash-based shell on Linux or [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/about). These commands copy the `osm` client binary to the standard user program location in your `PATH`.
+
+```console
+sudo mv ./linux-amd64/osm /usr/local/bin/osm
+sudo chmod +x /usr/local/bin/osm
+```
+
+For **macOS** use the following commands:
+
+```console
+sudo mv ./darwin-amd64/osm /usr/local/bin/osm
+sudo chmod +x /usr/local/bin/osm
+```
+
+You can verify the `osm` client library has been correctly added to your path and its version number with the following command.
+
+```console
+osm version
+```
+
+#### Windows
+
+In a PowerShell-based shell on Windows, use `Invoke-WebRequest` to download the OSM release and then extract with `Expand-Archive` as follows:
+
+```console
+# Specify the OSM version that will be leveraged throughout these instructions
+$OSM_VERSION="v0.9.1"
+
+[Net.ServicePointManager]::SecurityProtocol = "tls12"
+$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -URI "https://github.com/openservicemesh/osm/releases/download/$OSM_VERSION/osm-$OSM_VERSION-windows-amd64.zip" -OutFile "osm-$OSM_VERSION.zip"
```

Comment:
nit: in K8 there is an additional optional step to validate the checksum of the binary

Reply:
see https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/#install-kubectl-binary-with-curl-on-windows